src/Map/doc/index.rst

```diff
@@ -0,0 +1,211 @@
+        {
+            // 1. Create a new map instance
+            $myMap = (new Map());
+                ->center(new LatLng(46.903354, 1.888334))
+                ->zoom(6)
+            ;
+    
+            // 2. You can add markers, with an optional info window
+            $myMap
+                ->addMarker(new Marker(
+                    position: new LatLng(48.8566, 2.3522), 
+                    title: 'Paris'
+                ))
+                ->addMarker(new Marker(
+                    position: new LatLng(45.7640, 4.8357), 
+                    title: 'Lyon',
+                    // With an info window
+                    infoWindow: new InfoWindow(
+                        headerContent: '<b>Lyon</b>',
+                        content: 'The French town in the historic Rhône-Alpes region, located at the junction of the Rhône and Saône rivers.'
+                    )
+                ));
+    
+            // 3. And inject the map in your template to render it
+            return $this->render('contact/index.html.twig', [
+                'my_map' => $myMap,
+            ]);
+        }
+    }
+
+To render a map in your Twig template, use the ``render_map`` Twig function, e.g.:
+
+.. code-block:: twig
+
+    {{ render_map(my_map) }}
+    
+    {# or with custom attributes #}
+    {{ render_map(my_map, { style: 'height: 300px' }) }}
+
+Extend the default behavior
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony UX Map allows you to extend its default behavior using a custom Stimulus controller:
+
+.. code-block:: javascript
+
+    // assets/controllers/mymap_controller.js
+    
+    import { Controller } from '@hotwired/stimulus';
+    
+    export default class extends Controller {
+        connect() {
+            this.element.addEventListener('ux:map:pre-connect', this._onPreConnect);
+            this.element.addEventListener('ux:map:connect', this._onConnect);
+            this.element.addEventListener('ux:map:marker:before-create', this._onMarkerBeforeConnect);
+            this.element.addEventListener('ux:map:marker:after-create', this._onMarkerAfterCreate);
+            this.element.addEventListener('ux:map:info-window:before-create', this._onInfoWindowBeforeConnect);
+            this.element.addEventListener('ux:map:info-window:after-create', this._onInfoWindowAfterCreate);
+        }
+    
+        disconnect() {
+            // You should always remove listeners when the controller is disconnected to avoid side effects
+            this.element.removeEventListener('ux:map:pre-connect', this._onPreConnect);
+            this.element.removeEventListener('ux:map:connect', this._onConnect);
+            this.element.removeEventListener('ux:map:marker:before-create', this._onMarkerBeforeConnect);
+            this.element.removeEventListener('ux:map:marker:after-create', this._onMarkerAfterCreate);
+            this.element.removeEventListener('ux:map:info-window:before-create', this._onInfoWindowBeforeConnect);
```

Comment:
Just one more/last time ... i take my chance. 

I really do think _everything_ related to "solo infoWindows" should be removed/private/hidden for now :)

Reply:
See https://github.com/symfony/ux/pull/1937#discussion_r1678248193 
<img width="1017" alt="image" src="https://github.com/user-attachments/assets/d90d581c-5a01-49e1-bc9b-fccd1fd865bf">